seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/command/FlinkApiTaskExecuteCommand.java

```diff
@@ -46,7 +47,11 @@ public FlinkApiTaskExecuteCommand(FlinkCommandArgs flinkCommandArgs) {
     @Override
     public void execute() throws CommandExecuteException {
         Path configFile = FileUtils.getConfigPath(flinkCommandArgs);
-
+        if (!configFile.toFile().exists()) {
```

Comment:
All three type engine both use same FileUtils.getConfigPath, maybe you can put check logic in `FileUtils.getConfigPath`, you know, for redunce duplicate code.

Reply:
> All three type engine both use same FileUtils.getConfigPath, maybe you can put check logic in `FileUtils.getConfigPath`, you know, for redunce duplicate code.

Thank you for your suggestion, it is a good way.

Reply:
I have fixed it.